saleor/checkout/calculations.py

```diff
@@ -456,7 +458,7 @@ def _call_plugin_or_tax_app(
             plugin_ids=plugin_ids,
         )
         if not plugins:
-            raise TaxEmptyData("Empty tax data.")
+            raise TaxDataError("Empty tax data.")
```

Comment:
We can use here the new introduced `TaxDataErrorMessage`

Reply:
Thanks. As I mentioned in the PR description, plugin changes will come in separate PR.